extension/popup/popupView.jsx

```diff
@@ -480,8 +480,18 @@ const VoiceInput = ({ suggestions, onClickLexicon }) => {
   };
   return (
     <div id="voice-input">
-      {suggestions ? (
+      {buildSettings.inDevelopment ? (
```

Comment:
It looks like this is going to block suggestions except in development.

The `{suggestions ? ...` part needs to stay

Reply:
Alright, fixing that now.